tests/qam-updinstall/update_install.pm

```diff
@@ -230,7 +230,7 @@ sub run {
     foreach (@packages) {
         %bins = (%bins, get_packagebins_in_modules({package_name => $_, modules => \@modules}));
         # hash of hashes with keys 'name', 'supportstatus' and 'package'.
-        # e.g. https://smelt.suse.de/api/v1/basic/maintained/grub2
+        # e.g. https://%SMELT_URL%/api/v1/basic/maintained/grub2
```

Comment:
According to your change in [lib/maintenance_smelt.pm](https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/20520/files#diff-dbeb5f6d59e4d2f807aaa7358fe86cabd9ee06079dba3a941490887ac9f1bd80), the SMELT_URL should include the protocol.

```suggestion
        # e.g. %SMELT_URL%/api/v1/basic/maintained/grub2
```